pkg/analyzer/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: analyzer
-version: 0.29.6
+version: 0.29.6+1
```

Comment:
I guess it could/should be `.7`

There is no API change, but there is a behavior change.